src/components/custom/other-platforms.js

```diff
@@ -167,7 +167,7 @@ export const SmarttraderCard = ({ is_selected, word_break_cover }) => (
                     key={0}
                     translate_text="Trade the world’s markets on <0>Binary.com</0>’s classic platform."
                     components={[
-                        <LocalizedLinkText key={0} to="home" external="true" is_binary_link />,
```

Comment:
We can't use objects for this. This is because of the prop validation in `localize.js` on line 10 (which expects an argument with the type of array of nodes: 

```
Localize.propTypes = {
    components: PropTypes.arrayOf(PropTypes.node),
    translate_text: PropTypes.string,
    values: PropTypes.object,
}
```

Reply:
Sounds, good.